test/basic.expect.css

```diff
@@ -161,13 +161,13 @@
 
 .test-no-url {
 	order: 1;
-	background-image: img/test.png;
+	background-image: "img/test.png";
```

Comment:
this is related to a bug: https://github.com/csstools/postcss-image-set-function/issues/5